sheer/query.py

```diff
@@ -304,8 +306,23 @@ def get_document(doctype, docid):
         raw_results = es.get(index=es_index, doc_type=doctype, id=docid)
         return QueryHit(raw_results)
 
+    def convert_to_datetime(timestamp):
+        date = parser.parse(timestamp)
+        return date.astimezone(timezone('UTC'))
+
+    def when(starttime, endtime):
+        start = convert_to_datetime(starttime['date'])
```

Comment:
Doing it this way doesn't make too much sense. In other contexts, having the timezone was necessary so passing in the dictionary of the timestamp with the name of the timezone made sense.